content/conscious-coliving.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
Note we can use the nicer page layout including the image in frontmatter using:

```
layout: fancy
```

Just added this layout option (like blog but for non-blog pages).

Reply:
I've added the image in front matter and added this layout